ssh-keygentest.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from testutils import system
+ 
+class Firetest(unittest.TestCase):
+    def test_cat(self):
+        out,err,retcode=system('ssh-keygen -t rsa -f ~/.ssh/id_rsa -q -P ""')
```

Comment:
We are not using this `out, err, retcode` anywhere.
So just invoke the method instead in order to execute the bash command.
Replace the line by `system('ssh-keygen -t rsa -f ~/.ssh/id_rsa -q -P ""')`.